src/Tests/SenseNet.Workflow.Tests/WorkflowTestBase.cs

```diff
@@ -0,0 +1,279 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using IO = System.IO;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SenseNet.Configuration;
+using SenseNet.ContentRepository;
+using SenseNet.ContentRepository.i18n;
+using SenseNet.ContentRepository.Schema;
+using SenseNet.ContentRepository.Search;
+using SenseNet.ContentRepository.Security;
+using SenseNet.ContentRepository.Storage;
+using SenseNet.ContentRepository.Storage.Data;
+using SenseNet.ContentRepository.Storage.Data.SqlClient;
+using SenseNet.ContentRepository.Storage.Security;
+using SenseNet.Diagnostics;
+using SenseNet.Tests;
+using SenseNet.Tests.Implementations;
+using SenseNet.Workflow.Tests.Implementations;
+
+namespace SenseNet.Workflow.Tests
+{
+    public class WorkflowTestBase : TestBase
+    {
+        private static readonly string DatabaseName = "sn7tests_wf";
+
+        private static readonly string ConnectionStringBase =
+            @"Data Source=.\SQL2016;Integrated Security=SSPI;Persist Security Info=False";
+
+        private static readonly string ConnectionStringForWorkflowTests =
```

Comment:
It looks like either this property or the `GetConnectionString` method is unnecessary, because they do the same.